### PR TITLE
couple of SplashScreen names

### PR DESCRIPTION
**Pull request by LoganDark**
parameter names for renderProgressBar

image

names for these two fields

image

first contribution to yarn :D

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/SplashScreen.mapping [hunk 1]
```diff
@@ -7,12 +7,20 @@ CLASS net/minecraft/class_425 net/minecraft/client/gui/screen/SplashScreen
 	FIELD field_18219 reloading Z
 	FIELD field_18220 prepareCompleteTime J
 	FIELD field_2483 LOGO Lnet/minecraft/class_2960;
+	FIELD field_25041 bgArgb I
+	FIELD field_25042 bgRgb I
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/SplashScreen.mapping]:
```suggestion
	FIELD field_25041 BACKGROUND_ARGB I
	FIELD field_25042 BACKGROUND_RGB I
```

Comment (LoganDark) [reply mappings/net/minecraft/client/gui/screen/SplashScreen.mapping]: What's the benefit to making the names that long?

Comment (i509VCB) [reply mappings/net/minecraft/client/gui/screen/SplashScreen.mapping]: These are probably `static final` so captialized. Also `BACKGROUND` is much more explicit than `BG`

Also this is the color of the background?

Comment (LoganDark) [reply mappings/net/minecraft/client/gui/screen/SplashScreen.mapping]: @i509VCB It's the Mojang Studios brand color (the background of the splash screen). Maybe it should just be called `BRAND_[A]RGB` instead?

Edit: And yes, they are static final

**Comment by YanisBft** [comment]
@LoganDark mind update and resolve the comment please?

**Comment by LoganDark** [comment]
@YanisBft Sorry, I didn't get notifications for reactions

**Review by YanisBft — APPROVED** [review]

**Review by enbrain — APPROVED** [review]
It's still relevant in 20w46a.

**Comment by LoganDark** [comment]
what's the hold up? like wow

**Review by YTG1234 — APPROVED** [review]

**Pull request merged**